src/box/lua/net_box.lua

```diff
@@ -1231,7 +1384,7 @@ function remote_methods:unprepare(query, parameters, sql_opts, netbox_opts)
     if sql_opts ~= nil then
         box.error(box.error.UNSUPPORTED, "unprepare", "options")
     end
-    return self:_request(M_UNPREPARE, netbox_opts, nil, query, parameters or {},
+    return self:_request(M_UNPREPARE, netbox_opts, nil, self._stream_id, query, parameters or {},
```

Comment:
The line is too long (>80 characters). Please look through the diff and ensure that you don't introduce long lines.

Reply:
fixed